discovery-infra/test_infra/helper_classes/nodes.py

```diff
@@ -21,16 +21,11 @@ def __init__(self, node, cluster_host):
 class Nodes:
     DEFAULT_STATIC_IP_CONFIG = False
 
-    def __init__(self, node_controller: NodeController, private_ssh_key_path: Path):
+    def __init__(self, node_controller: NodeController):
```

Comment:
This one breaks `test_scaleup` 
FYI @lalon4 @nshidlin

Reply:
it also breaks our gathering of sos-report:
https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_assisted-test-infra/949/pull-ci-openshift-assisted-test-infra-master-e2e-metal-assisted/1414354554329239552/artifacts/e2e-metal-assisted/baremetalds-assisted-gather/build-log.txt